addOns/oast/src/main/java/org/zaproxy/addon/oast/OastService.java

```diff
@@ -120,4 +125,10 @@ public Thread newThread(Runnable r) {
             return t;
         }
     }
+
+    public String getPayload(boolean createNewPayload) throws Exception {
```

Comment:
@ricekot does this API change/behaviour look reasonable?

Reply:
okk thankyou

Reply:
Lets drop the existing payload and allow to paste only a new one. Sorry for the extra work.

Reply:
okay no problem so the new structure will look like : 
Insert New Payload -> Interachsh 
                                    Boast
                                    Callback

Reply:
@thc202 is there need to write test cases now? since I am not adding any functionality to oast service.

Reply:
Right, not needed.